tests/sample/ext/inc/sample_ext_helpers.h

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include <stdint.h>
+
+// Test Extension Hook program context.
+typedef struct _test_program_context
+{
+    uint8_t* data_start;
+    uint8_t* data_end;
+    uint32_t uint32_data;
+    uint16_t uint16_data;
+} test_program_context_t;
+
+// This file contains APIs for helper functions that are
+// exposed by the test extension.
+
+#define SAMPLE_EXT_HELPER_FN_BASE 0xFFFF
+
+#ifndef __doxygen
+#define EBPF_HELPER(return_type, name, args) typedef return_type(*name##_t) args
+#endif
+
+/**
+ * @brief Illustrates helper function with parameter of type EBPF_ARGUMENT_TYPE_PTR_TO_CTX.
+ * @param[in] context Pointer to program context.
+ * @retval 0. Illustrates EBPF_RETURN_TYPE_INTEGER.
```

Comment:
```suggestion
 * @retval 0 The operation was successful.
```
No period at end of first token.  retval creates a table row with first token in first column, rest in 2nd (description) column